Package.swift

```diff
@@ -14,99 +14,106 @@ let package = Package(
     .library(
       name: "Reporting",
       targets: [
-        "KSCrashFilters",
-        "KSCrashSinks",
-        "KSCrashInstallations",
+        Targets.filters,
+        Targets.sinks,
+        Targets.installations,
       ]
     ),
     .library(
       name: "Filters",
-      targets: ["KSCrashFilters"]
+      targets: [Targets.filters]
     ),
     .library(
       name: "Sinks",
-      targets: ["KSCrashSinks"]
+      targets: [Targets.sinks]
     ),
     .library(
       name: "Installations",
-      targets: ["KSCrashInstallations"]
+      targets: [Targets.installations]
     ),
     .library(
       name: "Recording",
-      targets: ["KSCrashRecording"]
+      targets: [Targets.recording]
     ),
   ],
   targets: [
     .target(
-      name: "KSCrashRecording",
+      name: Targets.recording,
       dependencies: [
-        "KSCrashRecordingCore"
+        .target(name: Targets.recordingCore)
       ],
+      resources: privacyResource,
       cSettings: [.headerSearchPath("Monitors")]
     ),
     .testTarget(
-      name: "KSCrashRecordingTests",
+      name: "\(Targets.recording)Tests",
```

Comment:
nit: Might be nice to have an extension on strings here and use `Targets.recording.tests`